### PR TITLE
Vlad dev

### PR DESCRIPTION
**Pull request by HornetPilot01**
Make Seeder

### CODE REVIEW

File: Bon_Voyage/DB/Entities/City.cs [hunk 1]
```diff
@@ -11,11 +11,13 @@ public class City
         [Key]
         public string Id { get; set; }
         public string Name { get; set; }
-        
+
+
```
Comment (Karonuk) [inline Bon_Voyage/DB/Entities/City.cs]: delete this

File: Bon_Voyage/Startup.cs [hunk 2]
```diff
@@ -103,6 +103,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseHttpsRedirection();
             
 
+            // Seeder
+            //SeederDB.SeedData(app.ApplicationServices, env, this.Configuration);
+
+
```
Comment (Karonuk) [inline Bon_Voyage/Startup.cs]: delete this

File: Bon_Voyage/ClientApp/.vscode/iisexpress.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "port": 42451,
+  "path": "f:\\Bon_Voyage\\Bon_Voyage\\ClientApp",
+  "clr": "v4.0",
+  "protocol": "http"
+}
```
Comment (Karonuk) [inline Bon_Voyage/ClientApp/.vscode/iisexpress.json]: this file should be deleted

File: Bon_Voyage/DB/SeederDB.cs [hunk 4]
```diff
@@ -0,0 +1,1194 @@
...
...ют, камеру схову багажу і перукарню. Цей 5-зірковий готель був відкритий 2007 році.\n\nРозташування\nRoyal Lagoons Aqua Park Resort \& Spa надає кімнати з системою опалення, кондиціонером і вбиральнею та має центральне розташування поблизу до Hurghada Grand Aquarium. Гості можуть дійти до Нова пристань для яхт, подолавши відстань 8 км звідси. До океанаріуму, барів і ресторанів можна дійти за кілька хвилин.\nНомери\nНомери виходять вікнами на сад.\n\nХарчування\nУ готелі щодня подають сніданок шведський стіл. Ресторан має широкий вибір фірмових делікатесів італійської кухні. Кафе-бар подає місцеві напої кожен день.\n\nПослуги\nRoyal Lagoons Aqua Park Resort \& Spa Хургада надає гостям цілодобовий ресепшн, цілодобове обслуговування номерів і послуги носія.\n\nВідпочинок\nГості можуть насолодитися приватним басейном з водною гіркою, гідромасажною ванною і сауною. Для занять спортом є спортзал, заняття аеробікою і фітнес-центр.\n\nІнтернет\nWi-Fi доступний в в номерах готелю за додаткову плату.\n\nWi-Fi надається в зонах загального користування готелю безкоштовно.\n\nПарковка\nНа території надається безкоштовна громадська парковка.\n\nКількість номерів:   366.".Replace(@"/n","<br />"),
```
Comment (Karonuk) [inline Bon_Voyage/DB/SeederDB.cs:582]: delete all \n and br

**Review by Maksym-Rohach — APPROVED** [review]

**Pull request merged**